server/container/guice/protocols/webadmin-mailbox/src/main/java/org/apache/james/modules/server/MailboxRoutesModule.java

```diff
@@ -71,18 +65,6 @@ protected void configure() {
         Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ONE_MAILBOX_TASKS));
         Multibinder.newSetBinder(binder(), TaskRegistration.class, Names.named(ONE_MAIL_TASKS));
 
-        bind(Zipper.class).in(Scopes.SINGLETON);
-        bind(ArchiveService.class).to(Zipper.class);
-
-        bind(ZipMailArchiveRestorer.class).in(Scopes.SINGLETON);
-        bind(MailArchiveRestorer.class).to(ZipMailArchiveRestorer.class);
-
-        bind(ZipArchivesLoader.class).in(Scopes.SINGLETON);
-        bind(MailArchivesLoader.class).to(ZipArchivesLoader.class);
-
-        bind(DefaultMailboxBackup.class).in(Scopes.SINGLETON);
-        bind(MailboxBackup.class).to(DefaultMailboxBackup.class);
-
         bind(ExportService.class).in(Scopes.SINGLETON);
         Multibinder.newSetBinder(binder(), TaskFromRequestRegistry.TaskRegistration.class, Names.named(UserMailboxesRoutes.USER_MAILBOXES_OPERATIONS_INJECTION_KEY))
             .addBinding().to(MailboxesExportRequestToTask.class);
```

Comment:
Regarding your JPA issues, it seems it's because JPA doesn't support the Blob module (so it breaks when trying to bind your export service). I think this mailboxBackup should not be supported for JPA.

You can then:
* move this binding for `ExportService` to your new module `MailboxesBackupModule`
* remove the install here and add the module `MailboxesBackupModule` directly in `MemoryJamesServerMain` and `CassandraJamesServerMain`

WDYT?

Reply:
It is not a good idea to ask for my opinion, since anything that runs without error is fine for a monkey in training :) 
Joking aside, may be we should ask the experts. @chibenwa Your thoughts on this?

Reply:
I propose adding yet another `MailboxExportRouteModule` with the routes declaration & the insall of `MailboxBackupModule` then Cassandra and memory can rely on it, and not JPA